services/libs/common_services/src/repos/llmPromptHistory.repo.ts

```
@@ -0,0 +1,31 @@
+import { DbStore, RepositoryBase } from '@crowd/database'
+import { Logger } from '@crowd/logging'
+import { ILlmResponse, LlmModelType, LlmQueryType } from '@crowd/types'
+
+export class LlmPromptHistoryRepository extends RepositoryBase<LlmPromptHistoryRepository> {
+  public constructor(dbStore: DbStore, parentLog: Logger) {
+    super(dbStore, parentLog)
+  }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)